content/docs/ai/ai-agents-workflow.md

```diff
@@ -0,0 +1,569 @@
+---
+title: Neon API for AI agents
+subtitle: Complete API workflow for database provisioning, auth, and versioning
+enableTableOfContents: true
+updatedOn: '2025-09-10T14:09:12.290Z'
+---
+
+With Neon's API, your agents can:
+
+- Provision PostgreSQL databases in ~500ms
+- Add production-ready authentication
+- Create database snapshots for version control
+- Implement per-user resource limits and usage tracking
+- Scale databases to zero when idle (no compute charges when idle; storage is still billed)
+
+Architecture assumption: This guide uses one Neon project per user for better isolation and security. For detailed architecture patterns and billing models, see the [platform integration getting started guide](/docs/guides/platform-integration-get-started).
+
```

Comment:
We need to cover the agent plan & how it works. 
- AI Platform signs up for paid account (30K) credit (gets a "paid" Org)
- Neon creates a second "Free" org for the account (for free Neon projects)
Then, we nee to describe how the platform can transfer a project from the Free org to the paid org if the user needs  higher usage limits for their app.
See: https://docs.google.com/document/d/1640OEFIKYLfWcTXw0-Zkf9kK0FzGnd8uR-VOjS4-_6g/edit?tab=t.0#heading=h.2tmyzq1po7pt